host/2.0/stretch/amd64/python-context/start.sh

```diff
@@ -3,47 +3,8 @@
 # Directory name for start.sh
 DIR="$(dirname $0)"
 
-# If we're not in a virtual environment, check if either:
-#   $AZURE_FUNCTIONS_VIRTUAL_ENVIRONMENT is set, use it for venv
-#   if it's not set and there is a default venv, activate it
-#   else use the container python
-if [ -z "$VIRTUAL_ENV"]
-then
-    # Determining the virtual environment entry point
-    if [ -z "$AZURE_FUNCTIONS_VIRTUAL_ENVIRONMENT" ]
-    then
-        echo "using system python"
-    else
-        echo "activating virtual environment"
-        source $AZURE_FUNCTIONS_VIRTUAL_ENVIRONMENT
-    fi
-fi
-
-if [ -z "$SKIP_PYTHONPATH_UPDATE" ]
-then
-    CUSTOM_PACKAGES="$HOME/site/wwwroot/.python_packages/lib/python3.6/site-packages"
-    if [ -d "$CUSTOM_PACKAGES" ]
-    then
-        echo "appending $CUSTOM_PACKAGES to PYTHONPATH"
-        export PYTHONPATH=$PYTHONPATH:$CUSTOM_PACKAGES
-    else
-        echo "path $CUSTOM_PACKAGES doesn't exist"
-    fi
-
-    CUSTOM_VENV_PACKAGES="$HOME/site/wwwroot/worker_venv/lib/python3.6/site-packages"
-    if [ -d "$CUSTOM_VENV_PACKAGES" ]
-    then
-        echo "appending $CUSTOM_VENV_PACKAGES to PYTHONPATH"
-        export PYTHONPATH=$PYTHONPATH:$CUSTOM_VENV_PACKAGES
-    else
-        echo "path $CUSTOM_VENV_PACKAGES doesn't exist"
-    fi
-else
-    echo "SKIP_PYTHONPATH_UPDATE == $SKIP_PYTHONPATH_UPDATE"
-fi
-
 echo "python == $(which python)"
 echo "PYTHONPATH == $PYTHONPATH"
 
 echo "starting the python worker"
-python $DIR/worker.py $@
\ No newline at end of file
+$HOME/site/wwwroot/worker/worker $@
```

Comment:
This will break existing applications that don't yet have the bundled worker. I'd rather add an if condition at the top for this file 

```
if [ -f $HOME/site/wwwroot/worker/worker_bundle ];
then 
....
```

also maybe rename it to `worker_bundle` so it;s a bit more unique. I'd also run a `chmod +x` on it to make sure the x permissions weren't dropped for whatever reason.

Reply:
basically if an app was published with an older cli, it should continue to work after this change.